tests/cert_without_extensions.rs

```diff
@@ -20,8 +20,7 @@ fn cert_without_extensions_test() {
     // `openssl x509 -in cert_without_extensions.der -inform DER -text -noout`
     const CERT_WITHOUT_EXTENSIONS_DER: &[u8] = include_bytes!("cert_without_extensions.der");
 
-    assert_eq!(
-        Some(webpki::Error::MissingOrMalformedExtensions),
```

Comment:
As you mention, I'd also suggest to rename "MissingOrMalformedExtensions" -> "MalformedExtensions"